content/20.pharmaceuticals.md

```diff
@@ -341,13 +335,12 @@ The first human monoclonal neutralizing antibody specifically against the SARS-C
 The 47D11 antibody clone was able to cross-neutralize SARS-CoV-1 and SARS-CoV-2.
 This antibody (now ABVV-47D11) has recently entered clinical trials in collaboration with AbbVie.
 Additionally, an extensive monoclonal neutralizing antibody pipeline has been developed to combat the ongoing pandemic, with over 50 different antibodies in clinical trials [@doi:10.1016/j.bsheal.2021.02.001].
-Thus far, two antibody cocktails (REGN-COV2 and LY-CoV555/LY-COV016) and the mAb Sotrovimab on its own have been granted an EUA by the FDA.
+Thus far, the mAb sotrovimab as well as two antibody cocktails (bamlanivimab/and estesevimab and casirivimab/imdevimab) have been granted EUA emergency use authorization by the FDA. <!---Note that there doesn’t seem to be anything about strovimab. If it's more recent than the others, maybe we can note that. I also changed the cocktails to include the generic drug names we use below, so it's easier to follow.-->
```

Comment:
```suggestion
Thus far, the mAb sotrovimab as well as two antibody cocktails (bamlanivimab/estesevimab and casirivimab/imdevimab) have been granted EUA emergency use authorization by the FDA. <!---Note that there doesn’t seem to be anything about strovimab. If it's more recent than the others, maybe we can note that. I also changed the cocktails to include the generic drug names we use below, so it's easier to follow.-->
```
I'll open a PR today to address strovimab and a couple other missing pieces that have been pointed out!